ISCSI/SCSI/SCSICommandDescriptorBlock/SCSICommandDescriptorBlock.cs

```diff
@@ -73,8 +75,14 @@ public static SCSICommandDescriptorBlock FromBytes(byte[] buffer, int offset)
                     return new SCSICommandDescriptorBlock16(buffer, offset);
                 case SCSIOpCodeName.ReportLUNs:
                     return new SCSICommandDescriptorBlock12(buffer, offset);
+                case SCSIOpCodeName.ReadDiscInformation:
+                    return new SCSICommandDescriptorBlock12(buffer, offset);
+                case SCSIOpCodeName.ReadToc:
+                    return new ReadTocCommand(buffer, offset);
+                case SCSIOpCodeName.GetConfiguration:
+                    return new GetConfigurationCommand(buffer, offset);
                 default:
-                    throw new UnsupportedSCSICommandException(String.Format("Unknown SCSI command: 0x{0}", opCode.ToString("x")));
+                    throw new UnsupportedSCSICommandException($"Unknown SCSI command: 0x{opCode:X}");

```

Comment:
If it ain't broke don't try to fix it please. while C# 2.0 may seem obsolete at this point, the entire project is compatible with it so I'm not sure I want to break it.

Reply:
Sure, I'll revert this change